src/current/_includes/releases/v24.3/v24.3.0-alpha.2.md

```diff
@@ -0,0 +1,88 @@
+## v24.3.0-alpha.2
+
+Release Date: October 14, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-0-alpha-2-security-updates">Security updates</h3>
+
+- The parameters for an [HBA config entry]({% link v24.3/security-reference/authentication.md %}#hba-configuration-syntax) for LDAP are now validated when the entry is created or amended, in addition to the validation that happens during an authentication attempt. [#132086][#132086]
+
+- Added automatic cleanup and validation for dropped roles when [altering default privileges]({% link v24.3/alter-default-privileges.md %}). [#131782][#131782]
+
+<h3 id="v24-3-0-alpha-2-general-changes">General changes</h3>
+
+- Changed the license `cockroach` is distributed under to the new CockroachDB Software License (CSL). [#131799][#131799] [#131794][#131794] [#131793][#131793]
+
+<h3 id="v24-3-0-alpha-2-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- You can now [authenticate to the DB console API]({% link v24.3/ui-overview.md %}#authentication) by supplying a Java Web Token (JWT) as a Bearer token in the Authorization header. [#130779][#130779]
```

Comment:
@pritesh-lahoti PTAL

Reply:
@mdlinville This feature is only required for JPMC right now and will be backported to 23.x.
We need not rush into documenting this feature publicly just yet. We could target it for 25.1.

Reply:
Hey, this is just for a release note -- you included release note text in your code commits, so it got picked up here by our automation. Are you asking me to remove this note? If so, you probably should not have included release note text in the commit, because each major version that gets the commit will have a draft release note that we will need to remove, and it's very likely that it will be a different writer who is working on that and will have no idea about this history.